### PR TITLE
Cancel pipeline run API

### PR DESCRIPTION
**Pull request by milantomic6**
## Description

Adding an API to cancel single pipeline run.

---

This checklist is used to make sure that common guidelines for a pull request are followed.

- [ ] **I have read the [contribution guidelines](https://github.com/Azure/azure-sdk-for-net/blob/AutoRest/.github/CONTRIBUTING.md).**
- [ ] **The pull request does not introduce [breaking changes](https://github.com/Azure/azure-sdk-for-net/blob/AutoRest/Documentation/breaking-changes.md).**

### [General Guidelines](https://github.com/Azure/azure-sdk-for-net/blob/AutoRest/.github/CONTRIBUTING.md#general-guidelines)
- [ ] Title of the pull request is clear and informative.
- [ ] There are a small number of commits, each of which have an informative message. This means that previously merged commits do not appear in the history of the PR. For more information on cleaning up the commits in your PR, [see this page](https://github.com/Azure/azure-powershell/blob/dev/documentation/cleaning-up-commits.md).

### [Testing Guidelines](https://github.com/Azure/azure-sdk-for-net/blob/AutoRest/.github/CONTRIBUTING.md#testing-guidelines)
- [ ] Pull request includes test coverage for the included changes.

### [SDK Generation Guidelines](https://github.com/Azure/azure-sdk-for-net/blob/AutoRest/.github/CONTRIBUTING.md#sdk-generation-guidelines)

https://github.com/Azure/azure-rest-api-specs/pull/1782#issuecomment-333989915

### CODE REVIEW

File: src/SDKs/DataFactory/Management.DataFactory/Generated/ActivityRunsOperationsExtensions.cs [hunk 1]
```diff
@@ -1,15 +1,15 @@
+// <auto-generated>
 // Copyright (c) Microsoft Corporation. All rights reserved.
 // Licensed under the MIT License. See License.txt in the project root for
 // license information.
 //
-// Code generated by Microsoft (R) AutoRest Code Generator.
+// Code generated by Microsoft (R) AutoRest Code Generator 2.2.22.0
```
Comment (bgold09) [inline src/SDKs/DataFactory/Management.DataFactory/Generated/ActivityRunsOperationsExtensions.cs]: How did you generate the code? I don't get these version numbers when I do it

Comment (milantomic6) [reply src/SDKs/DataFactory/Management.DataFactory/Generated/ActivityRunsOperationsExtensions.cs]: I ran "autorest" that is installed with npm, I believe I have the latest version.

Comment (bgold09) [reply src/SDKs/DataFactory/Management.DataFactory/Generated/ActivityRunsOperationsExtensions.cs]: how did you invoke it?

Comment (milantomic6) [reply src/SDKs/DataFactory/Management.DataFactory/Generated/ActivityRunsOperationsExtensions.cs]: I ran "autorest".

Comment (milantomic6) [reply src/SDKs/DataFactory/Management.DataFactory/Generated/ActivityRunsOperationsExtensions.cs]: I regenerated SDK based on latest instructions (use .md file as config source)

**Review by bgold09 — CHANGES_REQUESTED** [review]
Please bump the version to 0.2.1-preview and detail this change in changelog.md

File: src/SDKs/DataFactory/Management.DataFactory/changelog.md [hunk 2]
```diff
@@ -4,3 +4,8 @@
 
 ### Feature Additions
   * Initial public release of the Azure Data Factory V2 .NET SDK.
+
+## Version 0.2.1-preview
+
+### Feature Additions
+  * Cancel pipeline run api
```
Comment (bgold09) [inline src/SDKs/DataFactory/Management.DataFactory/changelog.md]: "Adding an API to cancel a pipeline run."

File: src/SDKs/DataFactory/Management.DataFactory/changelog.md [hunk 3]
```diff
@@ -4,3 +4,8 @@
 
 ### Feature Additions
   * Initial public release of the Azure Data Factory V2 .NET SDK.
+
+## Version 0.2.1-preview
```
Comment (bgold09) [inline src/SDKs/DataFactory/Management.DataFactory/changelog.md]: please move this section above the existing one, so the latest version is the first one from the top. thanks!

**Review by bgold09 — APPROVED** [review]

File: src/SDKs/DataFactory/Management.DataFactory/Microsoft.Azure.Management.DataFactory.csproj [hunk 4]
```diff
@@ -3,7 +3,7 @@
   <PropertyGroup>
     <PackageId>Microsoft.Azure.Management.DataFactory</PackageId>
     <Description>Azure Data Factory V2 is the data integration platform that goes beyond Azure Data Factory V1's orchestration and batch-processing of time-series data, with a general purpose app model supporting modern data warehousing patterns and scenarios, lift-and-shift SSIS, and data-driven SaaS applications. Compose and manage reliable and secure data integration workflows at scale. Use native ADF data connectors and Integration Runtimes to move and transform cloud and on-premises data that can be unstructured, semi-structured, and structured with Hadoop, Azure Data Lake, Spark, SQL Server, Cosmos DB and many other data platforms.</Description>
-    <VersionPrefix>0.2.0-preview</VersionPrefix>
+    <VersionPrefix>0.2.1-preview</VersionPrefix>
```
Comment (shahabhijeet) [inline src/SDKs/DataFactory/Management.DataFactory/Microsoft.Azure.Management.DataFactory.csproj]: @milantomic6 missing package release notes
Add < PackageReleaseNotes > and add Release notes.
I would recommend to use < ![CDATA and then format your Release notes

https://github.com/Azure/azure-sdk-for-net/blob/psSdkJson6/src/SDKs/MarketplaceOrdering/Management.MarketplaceOrdering/Microsoft.Azure.Management.MarketplaceOrdering.csproj#L10

**Review by shahabhijeet — CHANGES_REQUESTED** [review]
Missing scenario tests for the entire product. Look at the issue assigned to @bgold09.
Also I don't see metadata.txt file that is generated when you use generate.cmd to generate your code.
Also please add description that clearly describes the PR and the change you are making, also add link to the spec change PR.

**Comment by shahabhijeet** [comment]
@milantomic6 please reopen this PR once you have addressed the PR feedback.

**Comment by milantomic6** [comment]
Added scenario test.

**Pull request closed without merging**